src/Base/DrushCaller.php

```diff
@@ -115,13 +115,10 @@ public function sqlQuery($sql) {
     // in use.
     if (is_null($this->db_prefix)) {
       $sql_conf = $this->sqlConf('--format=json')->parseJson();
-
+      $this->db_prefix = isset($sql_conf->prefix) ? $sql_conf->prefix : NULL;
```

Comment:
I think it should be a blank string if not found. Setting it to NULL will force site-audit to check every database query to check the sql-conf again.